migtests/scripts/live-migration-fallf-run-test.sh

```diff
@@ -18,7 +18,7 @@ export REPO_ROOT="${PWD}"
 export SCRIPTS="${REPO_ROOT}/migtests/scripts"
 export TESTS_DIR="${REPO_ROOT}/migtests/tests"
 export TEST_DIR="${TESTS_DIR}/${TEST_NAME}"
-export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/export-dir"}
+export NORMALIZED_TEST_NAME="$(echo "$TEST_NAME" | tr '/-' '_')"
```

Comment:
where do we get `TEST_NAME` from to begin with?

Reply:
TEST_NAME is passed during the test invocation.
The general pattern is `TEST_SCRIPT TEST_NAME` 
example: `run_test.sh pg/datatypes`